cmd/juju/model/destroy.go

```diff
@@ -275,15 +268,6 @@ func (c *destroyCommand) Run(ctx *cmd.Context) error {
 	}
 	defer configAPI.Close()
 
-	// Check if the model has an SLA set.
-	slaIsSet := false
-	slaLevel, err := configAPI.SLALevel()
```

Comment:
There's a lot more moving parts to the SLA stuff. We should discuss how we proceed with it, with regards to whether we care about compatibility etc.

Reply:
Accept your point

But in this specific case, all this block of code does is set `slaIsSet` and `slaLevel` (unless configAPI.SLALevel() has some side-effect?), which are no longer used in this function, so can be safely removed right?